frontend/app/components/dropdown/dropdown.tsx

```diff
@@ -182,17 +183,16 @@ export default class Dropdown extends Component<Props, State> {
 
     return (
       <div className={b('dropdown', { mix }, { theme, active: isActive })} ref={this.rootNode}>
-        <Button
+        <UIButton
           aria-haspopup="listbox"
           aria-expanded={isActive && 'true'}
-          mix="dropdown__title"
-          type="button"
-          onClick={() => this.onTitleClick()}
-          theme="light"
-          className={titleClass}
+          onClick={this.onTitleClick}
+          theme={theme || 'light'}
+          mix={['dropdown__title', titleClass]}
```

Comment:
@Mavrin it is an example of `Array<string | undefined>` for `Mix` type. Sometimes you need to pass additional class name to an array that can be undefined.

Do you know why BEM method was chosen?

Reply:
Got it. 
I don't known why was chosen BEM.
I like css modules instead of BEM where it is possible. Or something like https://github.com/lttb/reshadow or https://github.com/4Catalyzer/astroturf.

Reply:
@Mavrin agree with you, css-modules are really good for any task. Also, I know another one css-in-js like lib without runtime https://github.com/callstack/linaria :)